admin/settings/class-abstract-settings.php

```diff
@@ -308,7 +308,11 @@ public function do_tabs_settings_section()
             );
 
             if (! empty($args['description']) ) {
-                echo esc_html($args['description']);
+                echo '<p>' . $args['description'] . '</p>';
```

Comment:
why did you remove `esc_html`? this is a security issue

Reply:
Didn't know that, adding!